### ISSUE TITLE
Syncing error - {'exercise_id': [u'This field cannot be blank.']}

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#4989]
## Summary

Trying to sync with central server gives an error

## Branch or installer method

0.16

## Traceback or relevant snippet from server.log
Nothing in Console or the server.log, errors on sync page:
```
Checking purgatory for unsaved models...
Purgatory is model-free! Congrats!
Initiating SyncSession...
Syncing models...
Exception uploading models (in api_client): ValidationError, , ()
Uploaded : 0 (0 failed, 1 error(s))
Downloaded : 0 (0 failed, 0 error(s))
Closing session... (Total uploaded: 0, Total downloaded: 0, Total errors: 1)
Checking purgatory once more, to try saving any unsaved models...
Purgatory is model-free! Congrats!
Upload error: {'exercise_id': [u'This field cannot be blank.']}
Completed with 1 errors.
```

## How to reproduce

1. Do some exercises/videos as Learner
1. Login as Admin and try to sync with central server

## Screenshots

![local_device_syncing](https://cloud.githubusercontent.com/assets/1457929/13706487/92351cac-e7a5-11e5-8f12-c1c50cfbebb4.png)

**Comment by jamalex** [comment]
Can you check whether an ExerciseLog or AttemptLog with empty exercise_id
exists locally. That shouldn't be the case, but would explain this.

Also, stuff involving the central server isn't quite ready to test; we're
still working on integrating the latest to there.

**Comment by rtibbles** [comment]
Fixed in #5039
